openreview/venue_request/process/deployProcess.py

```diff
@@ -30,12 +30,12 @@ def process(client, note, invitation):
 - Venue home page: {baseurl}/group?id={conference_id}
 - Venue Program Chairs console: {baseurl}/group?id={program_chairs_id}
 
-If you need to make a change to the information provided in your request form, please feel free to revise it directly using the "Revision" button. You can also control several stages of your venue by using the Stage buttons. Note that any change you make will be immediately applied to your venue.  
+If you need to make a change to the information provided in your request form, please feel free to revise it directly using the "Revision" button. You can also control several stages of your venue by using the Stage buttons. Note that any change you make will be immediately applied to your venue.
 If you have any questions, please refer to our FAQ: https://openreview.net/faq
 
 If you need special features that are not included in your request form, you can post a comment here or contact us at info@openreview.net and we will assist you.
 
-Best,  
+Best,
```

Comment:
Just one comment here: since markdown is enabled in comments now, I believe this will end up saying
Best, The OpenReview Team (in one line). I think adding two spaces or a `\` should fix it.

Reply:
Fixed